.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Contributing
+
+Thank you for your interest in contributing to the Internet Computer. By participating in this project, you agree to abide by our [Code of Conduct](./CODE_OF_CONDUCT.md).
```

Comment:
This doc needs some help. @lsgunnlsgunn did you say you had something in the works? I'd be happy to adapt it.

Reply:
I had a rough draft here:
https://github.com/dfinity-lab/documentation/blob/LG/DRAFT-contributing/CONTRIBUTING.md

Reply:
added a draft to this PR.